exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/otlp/traces/TraceRequestMarshalerTest.java

```diff
@@ -93,15 +103,20 @@ void toProtoResourceSpans() {
             InstrumentationLibrary.newBuilder().setName("testLib").setVersion("1.0").build());
   }
 
-  @Test
-  void toProtoSpan() {
+  static Stream<SpanContext> spanContexts() {
```

Comment:
I don't think we need two tests let's just stick with adding trace state to the existing